lms/djangoapps/certificates/views/webview.py

```diff
@@ -249,6 +250,10 @@ def _update_course_context(request, context, course, platform_name):
                                                               '{partner_short_name}.').format(
             partner_short_name=context['organization_short_name'],
             platform_name=platform_name)
+    fields = ['start', 'end', 'max_effort', 'language']
+    course_run_data = get_course_run_details(course_key, fields)
+    for field in fields:
+        context[field] = course_run_data[field]
```

Comment:
Since `get_course_run_details` only returns the requested fields, could you use https://docs.python.org/2.7/library/stdtypes.html#dict.update to update the context?

Reply:
Yes good call, thank you!